docs/faq.rst

```diff
@@ -257,7 +257,10 @@ Postfix, Dovecot, Nginx and Rspamd support overriding configuration files. Overr
 ``$ROOT/overrides``. Please refer to the official documentation of those programs for the
 correct syntax. The following file names will be taken as override configuration:
 
-- `Postfix`_ - ``postfix.cf`` in postfix sub-directory;
+- `Postfix`_ :
+   - ``postfix.cf`` as ``/overrides/postfix.cf``
+   - ``master.cf`` as ``/overrides/postfix.master``
+   - All ``/overrides/*.map`` files
```

Comment:
```suggestion
   - ``main.cf`` as ``$ROOT/overrides/postfix/postfix.cf``
   - ``master.cf`` as ``$ROOT/overrides/postfix/postfix.master``
   - All ``$ROOT/overrides/postfix/*.map`` files
```

I know that these are the paths mentioned in `start.py`, but from the perspective of the compose file and volumes the postfix overrides have their own subdirectory. It just gets mounted to `/overrides` within the resulting container.

Also the default name for `postfix.cf` within a postfix setup is `main.cf`, so we should probably call it that although the path is a different.

In the long tail it probably makes sense to rename these files and prevent further confusion, but documenting the current behavior as-is already helps very much :)

Reply:
hi @lub, I am fine with your suggested changes and will resolve this soon.